plugins/dell/fu-self-test.c

```diff
@@ -553,6 +553,7 @@ main (int argc, char **argv)
 	g_setenv ("FWUPD_UEFI_ESP_PATH", sysfsdir, TRUE);
 	g_setenv ("FWUPD_UEFI_TEST", "1", TRUE);
 	g_setenv ("FWUPD_DELL_FAKE_SMBIOS", "1", FALSE);
+	g_setenv("FWUPD_IGNORE_EFIVARFS", "1", FALSE);
```

Comment:
why a new variable?  can't we just flag the existing `FWUPD_UEFI_TEST`?

Reply:
Agreed, fixed.